xl/xdg.py

```diff
@@ -60,6 +60,8 @@
 if config_dirs is None:
     if sys.platform == 'win32':
         config_dirs = [exaile_dir]
+    elif 'freebsd' in sys.platform:
+        config_dirs = [data_dirs]
```

Comment:
This doesn't look right; `data_dirs` is already a list. Also, data dir is in `$prefix/share`, whereas we want `$prefix/etc`. The latest version&mdash;which I've just pushed&mdash;should handle the FreeBSD case correctly (it searches `/usr/local/etc` in addition to `/etc`), so this change shouldn't be necessary.